kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -1175,13 +1177,13 @@ define([
                 }
 
                 if (appInfo.app_output_param) {
-                    let outputName = file;
-                    // if we're in a subpath, need to strip it down to just the file name
-                    if (outputName.indexOf('/') !== -1) {
-                        outputName = outputName.substring(outputName.lastIndexOf('/') + 1);
+                    inputs[appInfo.app_output_param] = StringUtil.sanitizeWorkspaceObjectName(
```

Comment:
We were doing a simpler, incomplete, version of this here, now use the shiny new function instead!